ovs-persistence/src/main/java/org/dcsa/ovs/persistence/entity/Address.java

```diff
@@ -0,0 +1,57 @@
+package org.dcsa.ovs.persistence.entity;
+
+import lombok.Data;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import java.util.UUID;
+
+@Data
```

Comment:
If spring-data-jpa also supports private accessmodifier make the setters private to protect from unwanted state changes
(applies to all entities)

Reply:
I don't remember if it does tbh

Reply:
It seems people on the internet thinks it can access private fields like spring, so I'll change it

Reply:
Added

`@Builder`
`@NoArgsConstructor`
`@AllArgsConstructor`
`@Setter(AccessLevel.PRIVATE)`

The Setter makes them private, Builder so we can still create entities when testing, NoArgsConstructor since hibernate requires one and AllArgsConstructor since Builder requires one.